contracts/hyphen/LiquidityFarming.sol

```diff
@@ -84,6 +85,7 @@ contract HyphenLiquidityFarming is
     event LogRewardPoolInitialized(address _baseToken, address _rewardToken, uint256 _rewardPerSecond);
     event LogNativeReceived(address indexed sender, uint256 value);
     event LiquidityProviderUpdated(address indexed liquidityProviders);
+    event LogNftMigrated(address indexed owner, uint256 indexed nftId, IHyphenLiquidityFarmingV2 farmingContract);
```

Comment:
In the logs, IHyphenLiquidityFarmingV2 address will go or the whole interface?? I think we just need the address to be logged.

Reply:
Just the address would be logged, in these cases solidity automatically down casts interfaces to addresses.